applications/RomApplication/python_scripts/new_python_solvers_wrapper_rom.py

```diff
@@ -0,0 +1,61 @@
+import sys
```

Comment:
I recommend coming up with a better name rather than prepending with `new_` 
remember how long we were stuck with the `new_linear_solver_factory` :sweat_smile:

Reply:
:smile: Considering that this is a small research application I don't think we'll need to change keep backwards compatibility for a very long period of time. Indeed, the plan is to remove the old stuff ASAP. I'd keep the `new_` to constantly remind us to do it :smile:.